client/assets/images/customer-home/comment.svg

```diff
@@ -1 +1,8 @@
-<svg width="20" height="21" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M-2-1.5h24v24H-2z"/><path d="M18 15.67l-1.17-1.17H2v-12h16v13.17zM18 .5H2c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h14l4 4v-18c0-1.1-.9-2-2-2z" fill="#8e9196" fill-rule="nonzero"/></g></svg>
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+<mask id="mask0" mask-type="alpha" maskUnits="userSpaceOnUse" x="2" y="2" width="28" height="28">
```

Comment:
Non-blocking, but I wonder what icon editor we're using here. For simple icons I'm pretty sure this can be composed of pretty basic paths and shapes.

Reply:
I'll see if I can clean these up a bit. I left them masked so they're all a consistent 32x32 px square. Otherwise we get some alignment and sizing issues that need a bunch of extra CSS to fix.

Reply:
All of these except the Looka and add a page are actually Material icons, so maybe I'll just use the `MaterialIcon` component where we can instead of custom SVGs.